rt_gene/src/rt_gene/tracker_generic.py

```diff
@@ -101,10 +109,51 @@ def get_distance_matrix(self, new_elements):
                 distance_matrix[i][j] = self._tracked_elements[element_id].compute_distance(new_element)
         return distance_matrix, map_index_to_id
 
+    @staticmethod
+    def align_face_to_eyes(face_img, right_eye_center, left_eye_center, face_width=None, face_height=None):
+        # modified lightly from https://www.pyimagesearch.com/2017/05/22/face-alignment-with-opencv-and-python/
```

Comment:
We should add this to the list of libraries - it itself is adapted from https://github.com/MasteringOpenCV/code which allows free usage.

Reply:
Done